library-api/src/main/java/cz/muni/fi/pa165/library/facade/SingleLoanFacade.java

```diff
@@ -0,0 +1,36 @@
+package cz.muni.fi.pa165.library.facade;
+
+import cz.muni.fi.pa165.library.dto.BookDTO;
+import cz.muni.fi.pa165.library.dto.SingleLoanDTO;
+import cz.muni.fi.pa165.library.dto.SingleLoanDTO.UserDTO;
+
+import java.util.List;
+
+/** @author Martin Páleník 359817 */
+public interface SingleLoanFacade {
+
+    /** The time of loan will be set to now() */
+    public void borrowBook(BookDTO bookDto, UserDTO userDto);
+
+    /** The time of return will be set to now() */
+    public void returnBook(BookDTO bookDto, UserDTO userDto, String state);
+
```

Comment:
Use SingleLoanDto instead of those two dtos

Reply:
Thanks, could you please write a Facade for Loan so we can understand how these methods will be used?

Reply:
I'm on it.